aea/helpers/dialogue/base.py

```diff
@@ -241,17 +267,17 @@ def last_message(self) -> Optional[Message]:
             self.last_incoming_message is not None
             and self.last_outgoing_message is not None
         ):
-            last_incoming_message_id = cast(
-                int, self.last_incoming_message.get("message_id")
-            )
-            last_outgoing_message_id = cast(
+            last_outgoing_message_message_id = cast(
```

Comment:
I think we should add message_id as a property to message base class..there is no harm to that:

```python
@property
def message_id(self) -> int:
    assert self.is_set("message_id"), "message_id is not set!"
    return cast(int, self.get("message_id"))
```